### PR TITLE
Stethoscope vore shitpost

### PR DESCRIPTION
**Pull request by lbnesquik**
Make is so you can detect prey within people with your Stethoscope. ~~This word is awful to spell honestly.~~

### CODE REVIEW

**Comment by Screemonster** [comment]
reeeeee use "//vorestation edit" for single lines and "//vorestation edit begin" and "//vorestation edit end" for blocks if you're not editing a _vr file

File: code/modules/clothing/under/accessories/accessory.dm [hunk 1]
```diff
@@ -182,7 +196,7 @@
 								if(heart.is_bruised() || M.getOxyLoss() > 50)
 									sound = "[pick("odd noises in","weak")] heartbeat"
 								else
-									sound = "healthy heartbeat"
+									sound = "a healthy heartbeat"
```
Comment (SpadesNeil) [inline code/modules/clothing/under/accessories/accessory.dm]: // Vorestation Edit

File: code/modules/clothing/under/accessories/accessory.dm [hunk 2]
```diff
@@ -199,8 +213,12 @@
 								sound_strength = "hear a weak"
 								sound = "pulse"
 
-				user.visible_message("[user] places [src] against [M]'s [body_part] and listens attentively.", "You place [src] against [their] [body_part]. You [sound_strength] [sound].")
+				user.visible_message("[user] places [src] against [M]'s [body_part] and listens attentively.", "You place [src] against [their] [body_part]. You [sound_strength] [sound]. [message_holder]")
```
Comment (SpadesNeil) [inline code/modules/clothing/under/accessories/accessory.dm]: // Vorestation Edit

File: code/modules/clothing/under/accessories/accessory.dm [hunk 3]
```diff
@@ -199,8 +213,12 @@
...
 				return
+
+
+
+
```
Comment (SpadesNeil) [inline code/modules/clothing/under/accessories/accessory.dm]: Random white space?

Comment (lbnesquik) [reply code/modules/clothing/under/accessories/accessory.dm]: It's stylish.

File: code/modules/clothing/under/accessories/accessory.dm [hunk 4]
```diff
@@ -156,6 +156,20 @@
 
 /obj/item/clothing/accessory/stethoscope/attack(mob/living/carbon/human/M, mob/living/user)
 	if(ishuman(M) && isliving(user))
```
Comment (Arokha) [inline code/modules/clothing/under/accessories/accessory.dm]: Prevents shadekin from using it. What even is the point!!!

File: code/modules/clothing/under/accessories/accessory.dm [hunk 5]
```diff
@@ -156,6 +156,20 @@
...
+		//Vorestation edit start
+		var/message_holder	//Holds pervy message
+		var/beat_size = ""	//Small prey = quiet
+		for(var/belly in M.vore_organs) //Pervy edit.
+			var/obj/belly/B = belly
+			for(var/mob/living/carbon/human/H in B)
+				if(H.size_multiplier < 0.49)
```
Comment (Arokha) [inline code/modules/clothing/under/accessories/accessory.dm]: Seems dumb. Do < 0.50.

**Comment by lbnesquik** [comment]
Bump

File: code/modules/clothing/under/accessories/accessory.dm [hunk 6]
```diff
@@ -155,7 +155,21 @@
...
+		//Vorestation edit start
+		var/message_holder	//Holds pervy message
+		var/beat_size = ""	//Small prey = quiet
+		for(var/belly in M.vore_organs) //Pervy edit.
+			var/obj/belly/B = belly
+			for(var/mob/living/carbon/human/H in B)
+				if(H.size_multiplier < 0.5)
+					beat_size = pick("quiet ", "hushed " ,"low " ,"hushed ")
+				if(H.stat == DEAD)	//dead
+					message_holder = pick("Your listening is troubled by the occasional groaning of their body.", "There is some moderate burbling in the background.", "They seem to have a healthy metabolism as well.")
```
[the remark this answers is not in the document]
Comment (lbnesquik) [reply code/modules/clothing/under/accessories/accessory.dm]: Mhm. I could remove it and tack it on as an auxiliary thing for nutrition?

[the remark this answers is not in the document]
Comment (SpadesNeil) [reply code/modules/clothing/under/accessories/accessory.dm]: Do that, but like, nutrition over a certain amount.

[the remark this answers is not in the document]
Comment (lbnesquik) [reply code/modules/clothing/under/accessories/accessory.dm]: Sure

**Comment by lbnesquik** [comment]
@SpadesNeil Everything should be fine

File: code/modules/clothing/under/accessories/accessory.dm (hunk 4 above)
Comment (Arokha) [reply code/modules/clothing/under/accessories/accessory.dm]: On line 172 in your current edit, just print the belly messages you found so far and then return, if they are not a human. Everything below that is suitable only for humans. So change to isliving() at the top and then a small block for:
```
if(!ishuman(M))
    //do the message to whoever
    //return
```

**Comment by SpadesNeil** [comment]
so apparently this will runtime if someone tries to do it so fix it ASAP or I must revert this

**Pull request merged**